src/pages/home/home.module.css

```diff
@@ -0,0 +1,82 @@
+
+main{
+    padding: 0;
+}
+
+section{
+    padding: var(--padding-fluid-lg);
+}
+
+.hero{
+    position: relative;
+    height: 80vh;
+    background-image: var(--bg-gradient-overlay);
+    background-size: cover;
+    background-attachment: fixed;
+}
+/* 
+.hero::before{
+    content: '';
+    position: absolute;
+    inset: 0;
+    background-color: var(--tr-gray-overlay);
+    background-image: var(--bg-gradient-overlay);
+}
+
+.hero >*{
+    position: relative;
+    z-index: 1;
+} */
```

Comment:
remove unnecessary comment from code

Reply:
Thanks for pointing it out, fixed it in the latest commit